src/+replab/PermutationGroup.m

```diff
@@ -241,15 +241,17 @@
         end
 
     end
-    
+
     methods(Static)
-        %% Sign
 
         function sign = sign(perm)
-            %Returns sign of a given permuatation
-
-            %perm (row vector): Vector representing a permutation (e.g. [3 2 1 4])
-            %sign (float): sign of the permutation
+        % Returns the sign of a given permutation
+        %
+        % Args:
+        %   perm (permutation): Vector representing a permutation (e.g. [3 2 1 4])
```

Comment:
@Andrew-Buchanan-boop check the changes, in light of https://replab.github.io/replab/development/api_documentation_style.html#documenting-types